### ISSUE TITLE
CredDef creation inconsistently times out, doesn't always go to ledger

### ISSUE THREAD

**Issue by rblaine95** [didx-xyz/aries-cloudapi-python#560]
When creating a Credential Definition, sometimes it times out.
When trying to create the same Credential Definition, an error is returned saying that it already exists in the DB but not on the ledger.

```sh
$ response=$(curl -s -X 'POST' \
  "${apiBaseUrl}/generic/definitions/schemas" \
  -H 'accept: application/json' \
  -H "x-api-key: ${governanceApiKey}" \
  -H 'Content-Type: application/json' \
  -d @- <<EOF
{
  "name": "alpha",
  "version": "1.$i",
  "attribute_names": ["speed"]
}
EOF
)

$ schema_id=$(echo $response | jq -r '.id')

$ curl -s -X 'POST' \
  "${apiBaseUrl}/generic/definitions/credentials" \
  -H 'accept: application/json' \
  -H "x-api-key: ${issuer_token}" \
  -H 'Content-Type: application/json' \
  -d @- <<EOF
{
  "tag": "alpha",
  "schema_id": "${schema_id}",
  "support_revocation": false,
  "revocation_registry_size": 3000
}
```

**Comment by rblaine95** [comment]
It looks like it may be related to the Endorser <-> Webhooks connection

**Comment by rblaine95** [comment]
Theory:

1. On startup, Endorser opens a WS to the K8s Webhooks service.
2. The K8s service round-robins between the pods sitting behind it, so the WS connection goes to Webhooks-A.
3. User begins creating a CredDef, the event gets sent to Webhooks-B - the pod that Endorser _doesn't_ have an active WS connection to.
4. Eventually, CredDef creation times out.

**Comment by ff137** [comment]
Resolved by making websocket events stateless: #571
